crate/server/src/bootstrap_server/certificate.rs

```diff
@@ -5,11 +5,37 @@ use openssl::{
     nid::Nid,
     pkcs12::Pkcs12,
     pkey::{PKey, Public},
-    x509::X509Builder,
+    x509::{X509Builder, X509},
 };
+use ratls::generate_ratls_cert;
 
-use crate::result::KResult;
+use crate::{error, result::KResult};
 
+pub(crate) fn generate_ratls_pkcs12(
+    subject: &str,
+    expiration_days: u64,
+    pkcs12_password: &str,
+) -> KResult<Pkcs12> {
+    let (private_key, cert) = generate_ratls_cert(subject, vec![], expiration_days, None, true)
+        .map_err(|e| error::KmsError::RatlsError(e.to_string()))?;
+
+    std::fs::write("/tmp/cert.pem", &cert)?;
```

Comment:
I don't know if we need to write this cert.pem.

If so, should we use something like:

```
    let tmp_dir = TempDir::new()?;
    let tmp_path = tmp_dir.path();
```